### DISCUSSION TITLE
`flag.isConnected` for Embedded Boundaries

### DISCUSSION THREAD

**Discussion by etpalmer63** [AMReX-Codes/amrex#2797] [Q&A] [answered]
A user asks: 
> When using embedded boundaries (EB), how far out does
> `flag.isConnected(ii,jj,kk)` go? How does a cell ``(i,j,k)``
> know if a cell ``(i+1,j+1,k+1)`` is "connected" to it?

**Comment by WeiqunZhang** [comment, marked as answer]
The arguments to `isConnected(ii,jj,kk)` are either -1, 0, or 1.  The return value of `flag(i,j,k).isConnected(1,1,1)` can tell us if cell `(i,j,k)` is connected to cell `(i+1,j+1,k+1)`.  If we can walk through cell faces in no more than one step per direction from one cell to another, the two are considered `connected`.
